packages/stream-api/src/__tests__/streamApi.spec.ts

```diff
@@ -45,6 +46,16 @@ describe(StreamApi, () => {
     });
   });
 
+  describe('send createSceneAlteration', () => {
+    beforeEach(() => api.connect(url));
+
+    it('should send a request with Id', () => {
+      const request: vertexvis.protobuf.stream.ICreateSceneAlterationRequest = {};
```

Comment:
This type def shouldn't be necessary. Removes the need for an import and makes this line terser.